lib/web_ui/lib/src/engine/font_change_util.dart

```diff
@@ -23,13 +22,13 @@ FutureOr<void> sendFontChangeMessage() async {
   if (!_fontChangeScheduled) {
     _fontChangeScheduled = true;
     // Batch updates into next animationframe.
-    domWindow.requestAnimationFrame(allowInterop((num _) {
+    domWindow.requestAnimationFrame((_) {
```

Comment:
Not needing allowInterop here is pretty nice, NGL. If you were attempting to make the line between Dart and JS more clear, this is probably not helping :P

Reply:
Well, we still need `allowInterop`, but right now its happening behind the DOM shim. `requestAnimationFrame` isn't a JS function, it's a Dart function that calls `.toJS` on the function passed in. Also, `allowInterop` returns a Dart function so the line was pretty blurry even before this.

Look, honestly, if it were up to me I'd pepper the entire engine codebase with `toJS` / `toDart` and the DOM shim would just be external functions. But, this isn't my call, and honestly its probably easier for the migration to keep things hidden behind the shim when possible. I'm hoping in the future we do expose JS types beyond the shim so that it's clear when looking at code like this where we're calling out to JS.

However, in this particular case, would you prefer we move the `toJS` on Dart functions to the caller? I'm okay with that change.

Reply:
Nope, I think this is cool as is. I'm surprised the lint of "add types everywhere" is letting "_" slide without types, but I guess if it doesn't matter, it doesn't matter!